common/http/src/main/java/io/helidon/common/http/HeaderEnum.java

```diff
@@ -90,7 +90,10 @@ enum HeaderEnum implements Http.HeaderName {
     VARY("Vary"),
     WARNING("Warning"),
     WWW_AUTHENTICATE("WWW-Authenticate"),
-    X_HELIDON_CN("X-HELIDON-CN");
+    X_HELIDON_CN("X-HELIDON-CN"),
+    X_FORWARDED_FOR("X-Forwarded-For"),
+    X_FORWARDED_HOST("X-Forwarded-Host"),
+    X_FORWARDED_PROTO("X-Forwarded-Proto");
```

Comment:
`X-Forwarded-Port` also seems to be in use in the wild, in particular by OCI LBs. Here is a sanitized excerpt from what a Helidon server in OCI receives:

```
, x-forwarded-for=[w.x.y.z]
, x-forwarded-host=[a.b.c.com]
, x-forwarded-port=[443]
, x-forwarded-prefix=[/some/path]
, x-forwarded-proto=[https]
```

We probably need to declare that here and handle it in all the right places. (I won't tag each one with PR comments.)

Reply:
If I get it, we should not need x-forwarded-for, as that is the client (and we do not really care about the client).
We should care for the rest. If I get it, then a redirect should be:
```
{proto}://{host}:{port}/?{prefix}
```
(slash only inserted if prefix does not contain it). If any is missing, use the actual information from the request.
Is that correct?

Reply:
If the name prefix is correct, then it probably should be
`{proto}://{host}:{port}/?{prefix}/?{path}`

Reply:
I have added the missing headers to the constants. Not using them yet in the method to resolve original authority

Reply:
I did not say `port` and `prefix` were part of `Forwarded`. They are part of the non-standard `X-Forwarded-*` family. If `Forwarded` is not present but the `X-Forwarded-*` headers are then Helidon should use the ones that are present and that apply: proto, host, port, prefix.

`X-Forwarded-For` is in the example output only because it is part of what the OCI LBs seem to include. I did not mean to suggest that Helidon should use it for computing the authority or anything else we need for redirection or CORS decision-making.